### ISSUE TITLE
attempt to index field 'ITEMS' (a nil value)

### ISSUE THREAD

**Issue by oli356** [Ridepad/TrinketCDs#2]
```
Date: 2022-06-11 10:38:47
ID: 1
Error occured in: Global
Count: 1
Message: ..\AddOns\TrinketCDs\TrinketCDs.lua line 524:
   attempt to index field 'ITEMS' (a nil value)
Debug:
   (tail call): ?
   [C]: ?
   TrinketCDs\TrinketCDs.lua:524:
      TrinketCDs\TrinketCDs.lua:517
AddOns:
  KRT, v0.5.6b
  Swatter, v5.8.4723 (CreepyKangaroo)
  ACP, v3.2.0.118 
  ActionBarSaver, v
  AdvancedWotLKCombatLog, v
  AdvancedWotLKCombatLogHelper, v
  AtlasLoot, vv5.11.04
  AtlasLootFu, vv5.11.04
  Auctionator, v2.6.8
  BeanCounter, v5.8.4723 (CreepyKangaroo)
  BLT, v1.1
  DBMCore, v7.11
  DBMDefaultSkin, v
  DBMSpellTimers, v
  DBMStatusBarTimers, v
  DBMVPVEM, v11.0.2
  Decursive, v2.5.1
  ElvUI, v6.08
  ElvUIAddOnSkins, v1.06
  ElvUIEnhanced, v1.05
  ElvUIImprovedSpecSwitch, v1.00
  Enchantrix, v5.8.4723 (CreepyKangaroo)
  EnchantrixBarker, v5.8.4723 (CreepyKangaroo)
  FixGroups, v1.4.1
  GearScoreLite, v3x04
  KRU, v1.0.0
  LibGroupTalents10, v$Rev: 51 $
  LibTalentQuery10, v$Rev: 82 $
  LootCouncilLite, v1.011
  MageNuggets, v1.86
  Omen, v3.0.9
  Postal, v3.3.2
  RaidBuffStatus, vv3.274
  RaidWarningText, v
  SharedMedia, v3.1.0-191
  SharedMediaAdditionalFonts, v9.0.2
  Skada, v1.8.77
  SkadaImprovement, v
  Stubby, v5.8.4723 (CreepyKangaroo)
  SuperDuperMacro, v1.8.3
  TradeLog, v
  Transcriptor, v9.1.7
  TrinketCDs, v2.0.1
  BlizRuntimeLib_enUS v3.3.5.30300 <logon.warmane.com>
  (ck=455)


```

**Comment by Ridepad** [comment]
Thanks for report.
[Fixed in 2.0.2](https://github.com/Ridepad/TrinketCDs/releases/tag/2.0.2)

**Comment by oli356** [comment]
> Thanks for report. [Fixed in 2.0.2](https://github.com/Ridepad/TrinketCDs/releases/tag/2.0.2)

tyvm for quick fix
